firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionDetailsViewController.swift

```diff
@@ -0,0 +1,417 @@
+        button.setImage(UIImage(named: StandardImageIdentifiers.Medium.cross), for: .normal)
+    }
+    private var backButton: UIButton = .build { button in
+        button.layer.cornerRadius = 0.5 * TPMenuUX.UX.closeButtonSize
+        button.clipsToBounds = true
+        button.setTitle(.KeyboardShortcuts.Back, for: .normal)
+        button.setImage(UIImage(imageLiteralResourceName: StandardImageIdentifiers.Large.chevronLeft)
+            .withRenderingMode(.alwaysTemplate),
+                        for: .normal)
+        button.titleLabel?.font = TPMenuUX.Fonts.viewTitleLabels.scaledFont()
+    }
+
+    // MARK: Connection Status View
+    private let connectionView: UIView = .build { view in
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.layer.cornerRadius = TPMenuUX.UX.viewCornerRadius
+        view.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
+        view.layer.masksToBounds = true
+    }
+    private let connectionImage: UIImageView = .build { image in
+        image.contentMode = .scaleAspectFit
+        image.clipsToBounds = true
+        image.layer.masksToBounds = true
+        image.layer.cornerRadius = 5
+    }
+    private let connectionStatusLabel: UILabel = .build { label in
+        label.font = FXFontStyles.Regular.subheadline.scaledFont()
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.textAlignment = .left
+    }
+    private let dividerView: UIView = .build { _ in }
+    private var lockImageHeightConstraint: NSLayoutConstraint?
+
+    // MARK: Verified By View
+    private let verifiedByView: UIView = .build { view in
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.layer.cornerRadius = TPMenuUX.UX.viewCornerRadius
+        view.layer.maskedCorners = [.layerMinXMaxYCorner, .layerMaxXMaxYCorner]
+        view.layer.masksToBounds = true
+    }
+    private let verifiedByLabel: UILabel = .build { label in
+        label.font = FXFontStyles.Regular.subheadline.scaledFont()
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.textAlignment = .left
+    }
+
+    // MARK: See Certificates View
+    private lazy var viewCertificatesButton: LinkButton = .build { button in
+        button.titleLabel?.textAlignment = .left
+        button.titleLabel?.numberOfLines = 0
+        button.addTarget(self, action: #selector(self.viewCertificatesTapped), for: .touchUpInside)
+    }
+
+    // MARK: - Variables
+
+    private var constraints = [NSLayoutConstraint]()
+    var viewModel: TrackingProtectionDetailsModel
+    var notificationCenter: NotificationProtocol
+    var themeManager: ThemeManager
+    var themeObserver: NSObjectProtocol?
+    let windowUUID: WindowUUID
+    var currentWindowUUID: UUID? { return windowUUID }
+
+    // MARK: - View Lifecycle
+
+    init(with viewModel: TrackingProtectionDetailsModel,
```

Comment:
assuming this will be updated to use redux instead of view model in the future, let me know if I'm mistaken